BUILD.gn

```diff
@@ -3088,12 +3088,16 @@ skiasharp_build("SkiaSharp") {
   ]
 
   sources = [
+    // placed here to help seperate in PR
+    "src/xamarin/sk_managed_png_chunk_reader.cpp",
```

Comment:
Could the filename and the types not have underscores? Then it can be consistent and the generator can have a better time. Not sure how well it plays with underscores in the names. I know some work, but I may have some hidden rules somewhere that breaks it later on.

Reply:
maybe, i find underscores easier to read with long names

and if anything breaks in the generator we will just have to fix it as opposed to working around it :)

Reply:
We should still go with no underscores and all lowercase to be consistent as I know other people have generators for other languages.

Same with the one below - it is already PascalCase so it does not need underscores.